base/show.jl

```diff
@@ -1522,6 +1522,11 @@ recognize a variable, it uses a limited set of characters (greatly extended by
 Unicode). `isidentifier()` makes it possible to query the parser directly
 whether a symbol contains valid characters.
 
+The isidentifier function for symbols has been extended to include
+a normalization check. It ensures that the provided symbol is normalized
+using Unicode normalization with stability and composition enabled. This
+ensures consistent and standardized representation of the symbol.
```

Comment:
I don't think it is necessary to go into the details of the normalization — the standard Julia normalization is defined elsewhere in the documentation.  But it should be explained that this normalization corresponds to checking that the `Symbol` is equivalent to parsing a valid identifier string into a `Symbol`.   And it should be
put into a "compat" notice since this check is not performed in earlier versions.
```suggestion
!!! compat "Julia 1.11"
    In Julia 1.11 or later, `isidentifier` for `Symbol` arguments also ensures that the symbol is
    Unicode-normalized, so that it is equivalent to the result of parsing a valid identifier string
    into a `Symbol`.
```